server/src/util/logger.js

```diff
@@ -0,0 +1,42 @@
+const pino = require('pino');
+const pinoHttp = require('pino-http');
+const pinoPretty = require('pino-pretty');
+
+
+const fileStream = pino.destination('./output.log');
+
+const logger = pino({
+    transport:{
+        target: "pino-pretty",
+        options:{
+            translateTime: "SYS:dd-mm-yyyy HH:MM:ss",
+        }
+    },
+    redact: {
+        paths: ['req.headers'],
+        remove: true
+      }
+  }, fileStream);
+
+const loggerMiddleware = pinoHttp({
+    logger,
+    customLogLevel: function (res, err) {
+        // Customize the log level based on the response and error
+        return 'info';
+    },
+    customSuccessMessage: function (res) {
+        // Customize the success log message
+        return `Request completed successfully with status ${res.statusCode}`;
```

Comment:
I'm not sure why but I always get "Request completed successfully with status null"